documentation/blog/2023-06-22-svelte-dev-overhaul.md

```diff
@@ -0,0 +1,96 @@
+---
+title: 'svelte.dev: A complete overhaul'
+description: 'svelte.dev revamped, comes with accessibility fixes, new features and bottom navbar'
+author: Puru Vijay
+authorURL: https://puruvj.dev
+---
+
+It is June 2023, 4 years after svelte v3 was released. The then-practical-now-notorious single-pager website that folks are familiar with also came out then, and has stayed the same until now. Svelte's API surface has been increasing steadily and the single pager got bigger and bigger, to the point where it was becoming difficult to find things. The community has been asking for a revamp for a while now, and it's here now!
+
+Meet the new svelte.dev, a complete overhaul of the old website.
+
+## Multi-page docs
+
+The huge single-pager was a huge nuisance, in terms of finding stuff(Hullo there Ctrl+F), as well as just having low performance due to the sheer size of the page.
+
+We heard you! The docs are all split up into multiple pages now. And all pages are split into their sections in the sidebar. Wanna know how to use the `class:` directive, or wanna look up the syntax for `{#await}`? All these are just one click away under the `Template Syntax` heading in the sidebar.
+
+All modules exposed by svelte are also listed in the sidebar, under the `Runtime` section.
+
+- [svelte](/docs/svelte)
+- [svelte/store](/docs/svelte-store)
+- [svelte/motion](/docs/svelte-motion)
+- [svelte/transition](/docs/svelte-transition)
+- [svelte/animate](/docs/svelte-animate)
+- [svelte/easing](/docs/svelte-easing)
+- [svelte/action](/docs/svelte-action)
+
+And worry not, all the links from the older website will be redirected to the correct page.
+
+## Search
+
+One thing that the previous single pager website was good at was finding things without any JS. It's just a single html page, just hit <kbr>Ctrl+F</kbr> and search what you're looking for! But now that the site uses different pages, that is not an option anymore
```

Comment:
you mentioned above that finding stuff could be a nuisance. I think we could move that here
```suggestion
The lack of search functionality could make finding stuff a nuisance as <kbr>Ctrl+F</kbr> only returns results in order of occurrence and not order of importance. While <kbr>Ctrl+F</kbr> did have its benefits such as working without JS, now that the site has multiple pages, it's not an option anymore.
```